### PR TITLE
Edited adding image on Invest Arena [WAIV-1123]

### PR DESCRIPTION
**Pull request by konstantin1390**
Fixes #

### Changes

Summary of changes you made.

* Change 1
* Change 2

### Test plan

Explain how to test changes you made.

* [ ] Step 1
* [ ] Step 2

### Demo (optional)

Sreenshots (before & after) or video that show result of your work.

### CODE REVIEW

File: src/client/components/Comments/QuickCommentEditor.js [hunk 1]
```diff
@@ -137,9 +114,9 @@ class QuickCommentEditor extends React.Component {
             onPressEnter={this.handleSubmit}
             onChange={this.handleMsgChange}
           />
-          {_.isEmpty(image) && imageSelector}
+          {_.isEmpty(currentImage) && setImage}
         </div>
-        {Boolean(image) && (
+        {!_.isEmpty(currentImage) && (
```
Comment (milklim) [inline src/client/components/Comments/QuickCommentEditor.js]: import lodash method separately

File: src/client/components/ImageSetter/ImageSetter.js [hunk 2]
```diff
@@ -25,11 +62,58 @@ const ImageSetter = ({
...
+
+      onImageUpload(e.target.files[0], disableAndInsertImage, () => {
+        setLoadingImage(false);
+        onLoadingImage(false);
+      });
+    }
+  };
+
+  const handleRemoveImage = imageId => {
+    const filteredImages = currentImages.filter(f => f.id !== imageId);
```
Comment (milklim) [inline src/client/components/ImageSetter/ImageSetter.js]: why 'f' when we filter images? 'img => img.id !== imageId' looks better

File: src/client/components/Comments/QuickCommentEditor.js (hunk 1 above)
Comment (konstantin1390) [reply src/client/components/Comments/QuickCommentEditor.js]: ++

File: src/client/components/ImageSetter/ImageSetter.js (hunk 2 above)
Comment (konstantin1390) [reply src/client/components/ImageSetter/ImageSetter.js]: ++

**Pull request merged**